src/config/dark-sites.config

```diff
@@ -89,6 +89,8 @@ bondoer.fr
 bonsaihd.live
 botboy.snaz.in
 bouncingdvdlogo.com
+boxy-svg.com
```

Comment:
Please remove this entry 
![image](https://user-images.githubusercontent.com/25481501/110012386-6315eb00-7d18-11eb-8748-2f9e07378dc8.png)

Reply:
Oh strange, even in incognito they seem to be in dark mode by default on my end
![image](https://user-images.githubusercontent.com/48650812/110019894-dc194080-7d20-11eb-9e15-7405f2be197d.png)
and turning on the extension made practically no visible difference

Reply:
and also on all of the other sites